frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts

```diff
@@ -86,6 +86,7 @@ const checkAbsenceTypeSelectionButtonsExistance = async (
   arrivalTime = '08:15',
   departureTime = '16:00'
 ) => {
+  await page.reload()
```

Comment:
🤔

Reply:
Would a `page.goto` to the list page be more clear in intent? @kimifoud

Reply:
Should the fixtures be created before the actual test and first navigation?